drivers/ieee802154/ieee802154_b9x.c

```diff
@@ -6,7 +6,11 @@
 
 #define DT_DRV_COMPAT telink_b9x_zb
 
+#if CONFIG_SOC_RISCV_TELINK_B91 || CONFIG_SOC_RISCV_TELINK_B92
 #include "rf.h"
+#elif CONFIG_SOC_RISCV_TELINK_B95
+#include "lib/include/rf/rf_common.h"
```

Comment:
same Add "lib/include/rf" to CMake include list and revert this change

Reply:
will add the lib/include /rf into cmake list .